src/app/manageUsers/views/changeSubServiceToUser.ejs

```diff
@@ -0,0 +1,81 @@
+<div class="govuk-grid-row">
+
+    <div class="govuk-grid-column-two-thirds">
+
+        <!-- help page title and main content -->
+        <div id="make-someone-approver" class="help-section-title">
+            <h2 class="govuk-heading-l">
+                How to change a sub-service access on an end user's account
+            </h2>
+            <p class="govuk-body govuk-!-margin-right-6">
+                A sub-service is also known as a role or a form.
+            </p>
+            <p class="govuk-body govuk-!-margin-right-6">
+                As an approver, you can change an end user's sub-service access by adding or removing the sub-service from their account.
+            </p>
+        </div>
+
+        <!-- help page content -->
+        <p class="govuk-body">
+            To add or remove sub-service access from an edn user's account:
+        </p>
+        <ol class="govuk-list govuk-list--number">
+            <li>click on ‘Manage users’.</li>            
+            <li>Enter the name of the user in the search box whose access you want to change.</li>            
+            <li>Click on their name.</li>            
+            <li>Click the ‘Edit services’.</li>            
+            <li>Select the service the sub-service belongs to and click 'continue'.</li>            
+            <li>Select or deslect the sub-service that you want to add or remove and click 'continue'.</li>            
+            <li>Review the sub-service change for the end user and click 'approve'.</li>            
+            <li>A notification banner will confirm you have changed sub-service access for the end user.</li>
+        </ol>
+        <p class="govuk-body end-of-section">
+            An end user can also request changes to their sub-service access. Which an approver must either approve or reject.
+        </p>
+        <p class="govuk-body end-of-section">
+            Learn more about how to <a class="govuk-link" ref="#">approve a sub-service access request</a> and <a class="govuk-link" href="#">how to reject a sub-service request</a>.
```

Comment:
not sure how it should look? 
does it need a question mark in 'how to reject a sub-service request?'

Reply:
not according to designs its a link text I did add a location to go to when they press the button though